.ebextensions/04_install_ffmpeg.config

```diff
@@ -0,0 +1,11 @@
+commands:
+  01_become_root:
+    command: 'sudo su'
+  02_get_ffmpeg_executable:
+    command: 'wget -N https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-64bit-static.tar.xz -O ffmpeg-release-64bit-static.tar.xz'
```

Comment:
A public service providing more recent static builds of ffmpeg than the ffmpeg website itself does. Compiling this package from source takes unreasonably long.